CONTRIBUTING.md

```diff
@@ -45,6 +38,20 @@ Various other files are generated from the API specification. If you are running
 
 Larger changes will require updates to the documentation guides as well. This will be made clear during the code review.
 
+### TypeScript Type Generation
+
+If your changes involve `TypeScript types`, ensure to modify the corresponding files in [`utils/generate_types`](https://github.com/microsoft/playwright/blob/main/utils/generate_types).
```

Comment:
Wait, you are not supposed to edit `utils/generate_types` at all. Only the `docs/src`.